Content.Server/Administration/Rejuvenate.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Content.Server.GameObjects;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Administration
+{
+    class Rejuvenate : IClientCommand
+    {
+        public string Command => "rejuvenate";
+        public string Description
+        {
+            get
+            {
+                var localizationManager = IoCManager.Resolve<ILocalizationManager>();
+                return localizationManager.GetString("Fully heals a mob.");
```

Comment:
You really don't need to localize these either, IMO.

Reply:
It should break anything localizing these, right? I could see it being useful for someone localizing in the case of a server admin who doesn't speak english.

Reply:
I mean, true.